modules/planning/tasks/deciders/path_assessment_decider.cc

```diff
@@ -63,8 +63,8 @@ Status PathAssessmentDecider::Process(
   // 1. Remove invalid path.
   std::vector<PathData> valid_path_data;
   for (const auto& curr_path_data : candidate_path_data) {
-    // RecordDebugInfo(curr_path_data, curr_path_data.path_label(),
-    //                 reference_line_info);
+    RecordDebugInfo(curr_path_data, curr_path_data.path_label(),
+                    reference_line_info);
```

Comment:
This is for debugging purpose. Should comment this part in the commit. Otherwise, there will be too many paths printed on the simulator.

Reply:
ok